tests/org.eclipse.text.tests/projection/org/eclipse/text/tests/ProjectionDocumentTest.java

```diff
@@ -680,18 +679,18 @@ public void test9_7() {
 			return;
 
 		try {
-			int startOffset= fMasterDocument.getLineOffset(4);
-			Assert.assertEquals(80, startOffset);
-			int endOffset= fMasterDocument.getLineOffset(7);
-			Assert.assertEquals(140, endOffset);
+			int startOffset = fMasterDocument.getLineOffset(4);
+			Assertions.assertEquals(80, startOffset);
+			int endOffset = fMasterDocument.getLineOffset(7);
+			Assertions.assertEquals(140, endOffset);
 			fSlaveDocument.addMasterDocumentRange(startOffset, endOffset - startOffset);
 
 			assertSlaveContents(getOriginalMasterContents().substring(80, 140));
 
 			fMasterDocument.replace(endOffset, 1, "x");
 			assertLineInformationConsistency(fSlaveDocument);
 		} catch (BadLocationException e) {
-			assertTrue(false);
+			fail("BadLocationException occurred: " + e.getMessage());
 		}
```

Comment:
nitpicky: the try-catch could be removed in favor of having the test method throwing the exception, such that more information than only the message is provided to the test executor.